README.md

```diff
@@ -39,61 +39,72 @@ It's also possible to build and use a locally built image. See the
 The Docker ACAP can be run either unsecured or in TLS mode. The Docker ACAP uses
 TLS as default. Use the "Use TLS" dropdown in the web interface to switch
 between the two different modes. It's also possible to toggle this option by
-calling the parameter management API in [VAPIX](https://www.axis.com/vapix-library/)
-(accessing this documentation requires creating a free account) and setting the
-`root.dockerdwrapper.UseTLS` parameter to "yes" or "no".
+calling the parameter management API in [VAPIX](https://www.axis.com/vapix-library/) and setting the
```

Comment:
Since VAPIX is now open we could actually include a curl command here to change the parameter. That doesn't have to be done in the present PR though - just a comment.

Reply:
I think it's a good comment, I've updated to documentation to include the curl command to enable TLS.